build/configs/rtl8730e/rtl8730e_download.sh

```diff
@@ -91,16 +91,18 @@ function pre_download()
 		if [[ "${parts[$partidx]}" == "ss" ]];then
 			continue
 		fi
-		LAST_PARTITION=${parts[$partidx]}
+		LAST_OFFSET=${offsets[$partidx]}
 	done
-	LAST_IMAGE=${BOOTPARAM}.bin	
+	LAST_IMAGE=${BOOTPARAM}.bin
 
 	if [[ -n ${CONFIG_ARCH_BOARD_HAVE_SECOND_FLASH} ]];then
 		source ${TOP_PATH}/build/configs/${CONFIG_ARCH_BOARD}/board_metadata.txt
 		if test -f "${BIN_PATH}/${EXTERNAL}.bin"; then
 			cp -p ${BIN_PATH}/${EXTERNAL}.bin ${IMG_TOOL_PATH}/${EXTERNAL}.bin
 		fi
 		LAST_IMAGE=${RESOURCE_BIN_NAME}
+		LAST_OFFSET=0xAEC0000
```

Comment:
What is this for? It looks like its hardcoded.

Reply:
I update it, thanks.